rome-modules/src/main/java/com/rometools/modules/itunes/io/ITunesParser.java

```diff
@@ -92,7 +92,7 @@ public String getNamespaceUri() {
                     if (subcategory != null && subcategory.getAttribute("text") != null) {
                         final Subcategory subcat = new Subcategory();
                         subcat.setName(subcategory.getAttribute("text").getValue().trim());
-                        cat.setSubcategory(subcat);
+                        cat.addSubCategory(subcat);
```

Comment:
Looks like it's only parsing the first subcategory unlike the generator which generates a list of subcategories?